src/qml/AppSwitcher.qml

```diff
@@ -90,8 +92,8 @@ Item {
                 }
 
                 delegate: Item {
-                    width: (flickable.width - (gridview.spacing * gridview.columns)) / gridview.columns
-                    height: width * (desktop.height / desktop.width)
+                    width: (desktop.width-(gridview.spacing * gridview.columns)) / gridview.columns
```

Comment:
there are mix space styles

looks like correct is `flickable.width - (gridview.spacing *`